<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Employee</class>
 <widget class="QDialog" name="Add_Employee">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Age</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>User_ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>emp_ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="first_name_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="last_name_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="age_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="profile_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="user_ID_lineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="password_lineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="emp_ID_lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="add_pushButton">
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
